uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChangeTest.php

```diff
@@ -794,6 +794,21 @@ public function testTheLeaveRequestBreakdownReturnsOnlyTheLeaveBalanceChangesOfT
     }
   }
 
+  public function testTheLeaveRequestBreakdownReturnsEmptyArrayForSoftDeletedLeaveRequests() {
```

Comment:
A better name for this would be "testTheLeaveRequestBreakdownShouldNotReturnSoftDeletedLeaveRequests"